CHANGELOG.md

```diff
@@ -2,13 +2,41 @@
 
 ## Unreleased
 
+## v0.99.0
+
 ### 🛑 Breaking changes 🛑
 
 - (Splunk) `receiver/discovery`: Update the component to emit entity events 
   - Emit entity events for discovered endpoints with log_endpoints: true ([#4684](https://github.com/signalfx/splunk-otel-collector/pull/4684))
   - Ensure active endpoints emitted as entity events periodically ([#4684](https://github.com/signalfx/splunk-otel-collector/pull/4684))
   - Emit entity events only for matching receivers ([#4691](https://github.com/signalfx/splunk-otel-collector/pull/4691))
   - Remove `log_endpoints` config option ([#4692](https://github.com/signalfx/splunk-otel-collector/pull/4692))
+- (Core)`builder`: Add strict version checking when using the builder. Add the temporary flag `--skip-strict-versioning `for skipping this check. ([#9896](https://github.com/open-telemetry/opentelemetry-collector/pull/9896))
+    Strict version checking will error on major and minor version mismatches
+    between the `otelcol_version` configured and the builder version or versions
+    in the go.mod. This check can be temporarily disabled by using the `--skip-strict-versioning`
+    flag. This flag will be removed in a future minor version.
+- (Core) `telemetry`: Distributed internal metrics across different levels. ([#7890](https://github.com/open-telemetry/opentelemetry-collector/pull/7890))
+  The internal metrics levels are updated along with reported metrics:
+  - The default level is changed from `basic` to `normal`, which can be overridden with `service::telmetry::metrics::level` configuration.
+  - Batch processor metrics are updated to be reported starting from `normal` level:
+    - `processor_batch_batch_send_size`
+    - `processor_batch_metadata_cardinality`
+    - `processor_batch_timeout_trigger_send`
+    - `processor_batch_size_trigger_send`
+  - GRPC/HTTP server and client metrics are updated to be reported starting from `detailed` level:
+    - http.client.* metrics
+    - http.server.* metrics
+    - rpc.server.* metrics
+    - rpc.client.* metrics
```

Comment:
We exclude all these metrics in our distro by default. We need to mention that here I believe:

```
All these metrics were already excluded in our distro by default. So this change only affects users who reconfigured the default dropping rules (`metric_relabel_configs`) in the prometheus receiver scraping internal metrics.
```